src/components/AstronautsPage/api/services/fetchAstronauts.ts

```diff
@@ -1,12 +1,13 @@
 import axios from 'axios';
 import { Endpoints } from '../../../../shared/api/constants/endpoints';
 
-export const fetchAstronauts = async () => {
+export const fetchAstronauts = async (limit: number, page: number) => {
+  const offset = (page - 1) * limit;
```

Comment:
why do you calculating this offset?

Reply:
Because the loading of the next pack of astronauts depends on the offset number. And the offset number depends on the page. And the number of the offset depends on the page and the limit of cards on the page.